packages/core/src/components/search/SearchProductItem/SearchProductItem.tsx

```diff
@@ -50,6 +68,12 @@ function SearchProductItem({
     onClick: () => {
       onClick()
       onSearchSelection?.(name, href)
+      sendAutocompleteClickEvent({
+        url: href,
+        term: name,
+        position: index,
+        productId: product.isVariantOf.productGroupID ?? product.sku,
```

Comment:
Applying the same logic used [here](https://github.com/vtex/faststore/blob/6f2e5b5970de162e5c425f70ce20552fe195a3de/packages/core/src/sdk/analytics/platform/vtex/search.ts#L114)